openpype/hosts/houdini/api/creator_node_shelves.py

```diff
@@ -196,7 +196,9 @@ def install():
 
             key = "openpype_create.{}".format(identifier)
             log.debug(f"Registering {key}")
-            script = CREATE_SCRIPT.format(identifier=identifier)
+            script = CREATE_SCRIPT.format(
+                identifier=identifier, variant=creator.get_default_variant()
```

Comment:
yeah well that's because I removed the logic of where `default_variant` is coming from on this PR after @iLLiCiTiT addressed it on another PR (https://github.com/ynput/OpenPype/pull/5436). I would expect if we merge `develop` over now this should be fixed

Reply:
I have merged latest `develop` to this branch now